src/JsonPath/Expressions/InOperator.cs

```diff
@@ -31,6 +31,6 @@ public bool Evaluate(PathValue? left, PathValue? right)
 
 	public override string ToString()
 	{
-		return "<";
+		return "in";
```

Comment:
Can you add a test in a new test class called `StringifyTests`, please?  I'll fill in others that are missing afterward.

Reply:
I think this may need surrounding spaces:

```suggestion
		return " in ";
```

To be confirmed with a test, but I would expect the syntax to require them.

Reply:
@gregsdennis I think I may not be able to ,because I'm a daily Scala/Java user, sorry for that.